builtins-default-costs/src/lib.rs

```diff
@@ -15,8 +13,7 @@ use {
 };
 
 #[derive(Clone)]
-#[cfg_attr(feature = "svm-internal", qualifiers(pub))]
-struct MigratingBuiltinCost {
+pub struct MigratingBuiltinCost {
```

Comment:
Is it necessary to make this, and the rest of changes, `pub`? They could remain private.